src/tests/Interop/ObjectiveC/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+set(CMAKE_OBJC_STANDARD 11)
```

Comment:
Should we also set the minimum cmake version? You had mentioned that the built-in Objective-C support required something higher than our current official minimum.